kwok/commands.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
I'd rather you find a way to do the cluster creation and image build time and not at image start time.

Reply:
Done